schema/artist/index.js

```diff
@@ -71,12 +73,14 @@ const ShowField = {
     sort: PartnerShowSorts,
   },
   resolve: ({ id }, options, request, { rootValue: { relatedShowsLoader } }) => {
+    // TODO: Fix upstream, for now we remove shows from certain Partner types
     return relatedShowsLoader(
       defaults(options, {
         artist_id: id,
         sort: "-end_at",
+      })).then(shows => {
+        return reject(shows, show => includes(blacklistedPartnerTypes, show.partner.type))
```

Comment:
#minor Does doing this rejection after the resolving mean that counts might get out of sync, like if you supplied a `size` param for shows?

Reply:
Yup!

However, I'm not sure of any client use that does that type of scrolling/pagination, and would get thrown off by that.

Force for instance, doesn't, it just requests and stitches together several of these to render the full CV (or highlights). So it would be unaffected by that.